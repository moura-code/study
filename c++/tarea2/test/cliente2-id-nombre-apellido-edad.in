crearCliente 31 Valquiria Sanchez 12
imprimirCliente
imprimirEdadCliente
imprimirIdCliente
liberarCliente
Fin
