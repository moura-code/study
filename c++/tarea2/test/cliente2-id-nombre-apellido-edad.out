Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Cliente creado de forma exitosa.
2> Cliente Valquiria Sanchez
Id: 31
Edad: 12
3> La edad del cliente es: 12
4> El id del cliente es: 31
5> Cliente liberado con exito.
6> Fin.
